### ISSUE TITLE
:root selector fails in isolated component

### ISSUE THREAD

**Issue by whitecolor** [cyclejs/cyclejs#329]
http://www.webpackbin.com/NJS8razEZ

``` js
// inside isolated component
DOM.select(':root')
```

gives an error:

```
Failed to execute 'matches' on 'Element': '' is not a valid selector.
```

this is related: https://github.com/cyclejs/core/issues/310

cc @TylorS

**Comment by TylorS** [comment]
Which version are you using that this happens on?

**Comment by whitecolor** [comment]
The last published rc-31.

**Comment by jemmyw** [comment]
@TylorS I'm also getting this with rc-31. In EventDelegator matchEventAgainstDestinations the passed selector is blank.

**Comment by staltz** [comment]
Trying to understand this issue: what do you expect `select(':root')` to select? 

- The highest element **inside** the current isolated component 
- or
- The container element given to `makeDOMDriver()`

**Comment by whitecolor** [comment]
I assume `select(':root')` should be the the highest element in the context of the component, for isolated it is highest element inside the current isolated component.

**Comment by raquelxmoss** [comment]
Is this issue ready to be worked on? I'd like to give it a go if no one else is.

**Comment by TylorS** [comment]
This could possibly be fixed already with some improvements since rc31. A good place to start would be adding a test, it would be useful either way.

@raquelxmoss I'd say it is in fact ready

**Comment by staltz** [comment]
Down grading to Priority 1 until we have a test for this.

@raquelxmoss perhaps?

**Comment by staltz** [comment]
I think this was fixed in #310.
